packages/react/src/VideoPlayer/VideoPlayer.features.stories.tsx

```diff
@@ -11,14 +12,85 @@ export default {
 
 export const WithPoster = () => (
   <VideoPlayer poster={posterImage} title="GitHub media player">
-    <VideoPlayer.Source src="/example.mp4" type="video/mp4" />
-    <VideoPlayer.Track src="/example.vtt" />
+    <VideoPlayer.Source src="https://primer.github.io/brand/assets/example.mp4" type="video/mp4" />
+    <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
   </VideoPlayer>
 )
 
 export const WithoutBranding = () => (
-  <VideoPlayer branding={false} title="GitHub media player">
-    <VideoPlayer.Source src="/example.mp4" type="video/mp4" />
-    <VideoPlayer.Track src="/example.vtt" />
+  <VideoPlayer showBranding={false} title="GitHub media player">
+    <VideoPlayer.Source src="https://primer.github.io/brand/assets/example.mp4" type="video/mp4" />
+    <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
+  </VideoPlayer>
+)
+
+export const WithoutTitle = () => (
+  <VideoPlayer showTitle={false} title="GitHub media player">
```

Comment:
these props `showTitle={false} title="GitHub media player"` appear to be a little at odds with one another. 

I'm inferring that we need to always set a title, but `showTitle` hides it visually? If that's the case, could we change the prop name of `showTitle` to `visuallyHiddenTitle`? We'd need to capture it in the changeset, but `visuallyHidden` is the naming we use for this type of behaviour in Primer.

Reply:
Great suggestion, thank you

Reply:
Resolved in b1d23e389a959cc9ff4802e07f9bc05645f7771d